tasks/random_dot_motion.md

```diff
@@ -0,0 +1,43 @@
+# Summary
+> Participants are asked to answer which direction (0 to 360 degrees) a circular field of 50 dots are moving.
+
+# References
+> Dynamics of judgment propagation. Mehdi Moussaïd, Stefan M. Herzog, Juliane E. Kämmer, Ralph Hertwig. Proceedings of the National Academy of Sciences Apr 2017, 114 (16) 4117-4122; DOI: 10.1073/pnas.1611998114
+
+# Stimuli
+## The visual components
+> Video of dot movement: http://movie-usa.glencoesoftware.com/video/10.1073/pnas.1611998114/video-1
+
+## Materials for alternative versions of the experiment 
+> There are 2 versions of the study. In version 1,  2 individuals were paired. In study 2, 6 participants were paired in a chain.
+
+# Procedure
+## Steps
+> There are three difficulty levels. The three difficulty levels were characterized by
+different proportions of correlated dots, pc = 50%, 30%, and 5%. In addition, the deviation was set to 0° in difficulty level 1 and to 45° in
+difficulty levels 2 and 3.
+
+ > There are 2 versions of the study
+ > Version 1: Individuals were paired with the same partner for 15 consecutive rounds. In each round, the individual made a guess, then looked at their partner's estimate, and finally revised their guess. Partners always had the same correct angle programmed into the dots, but the difficulty level could vary by trial between the pair. 
+ > Version 2: A group of 6 participants (A, B...F) are connected along a chain. After each initial guess, A's estimate is shown to B, B's estimate is shown to C, ... E's estimate is shown to F. This was repeated across 15 rounds. A (head of chain) always experienced difficultu level 1, and all other participants in the chain experinced difficulty level 3. 
+
+## Details of Task development
+> The task was developed using HTML/JavaScript and SmartFoxServer. A set of 25 unique angles equidistantly distributed between 0° and
+360° served as true angles. The animation was displayed at a rate of 20 frames
+per s.
+
+
+## Instructions
+> Complete instructions shown to participants at each stage of the task.  
+> Write the complete instructions used from the start to the end of the experiment. This should include any instruction provided on each of the steps. If instructions have step interdependence, or are role specific, describe that in full detail.
+> 1. While the dots are still moving, participants indicated their estimate by using the computer mouse to place a black arrow in the circular area.
+> 2. After confirming the estimate (by a mouse click), the dots stopped moving and a blue arrow indicating the partner's guess was displaced. Participants were instructed to revise (or not revise) their estimate, and confirm their new estimate.
+> 3. The correct angle was displaced as a red arrow. 
+
+
+# Criteria
+## Performance calculation
+> A points system was used to reward more accurate guesses. The function $p=10^{-5e_f^2}$, where $e_f$ is the the error on the final estimate in radians, was rounded to the nearest integer. 
```

Comment:
Should we add in the range of points they could get? Participants could score between 0-10 points per round.